backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/BpkPrice.kt

```diff
@@ -43,6 +45,7 @@ fun BpkPrice(
     leadingText: String? = null,
     previousPrice: String? = null,
     trailingText: String? = null,
+    previousPriceColor: Color? = BpkTheme.colors.textSecondary,
```

Comment:
this shouldn't be a configurable option, but be `textError` for all

Reply:
Don't we need to pass the`previousPriceColor` parameter from outside? The default value is`textError`, right?